pyproject.toml

```diff
@@ -1,6 +1,58 @@
-[tool.black]
-line-length = 80
-target-version = ['py38']
+[project]
+name = "ec-api-proxy"
+version = "0.0.0"
+description = "White label consumer of DC APIs"
+readme = "README.md"
+requires-python = "==3.12.*"
+dependencies = [
+    "django==4.2.22",
+    "sentry-sdk==2.28.0",
+    "django-pipeline==2.1.0",
+    "django-debug-toolbar==4.2.0",
```

Comment:
Ideally I would have liked to move django-debug-toolbar to the dev group, but it requires some faffing about with the settings. I decided not to scope creep this one today, so I just left it.

Reply:
Worth an issue/asana ticket?

Reply:
done https://app.asana.com/1/1204880536137786/project/1204880927741389/task/1210556779655486?focus=true